core_tic_tac_toe/human_player.py

```diff
@@ -1,9 +1,10 @@
 class HumanPlayer:
 
-    def __init__(self, name, symbol, board):
+    def __init__(self, name, symbol, board, io):
```

Comment:
Discussed injecting i/o into display, and then injecting display into player, since display's responsibility is to display things to the console. Design choice is not set in stone though.